stanford-sw/src/edu/stanford/Item.java

```diff
@@ -94,7 +94,8 @@ public Item(DataField f999, String recId) {
 					|| StanfordIndexer.SKIPPED_LOCS.contains(homeLoc)
 					|| itemType.equals("EDI-REMOVE")
 					// SW-849 skip PHYSICS items that aren't location PHYSTEMP
-					|| (library.equals("PHYSICS") && (!homeLoc.equals("PHYSTEMP") && !currLoc.equals("PHYSTEMP"))))
+					|| (library.equals("PHYSICS") && (!homeLoc.equals("PHYSTEMP") && !currLoc.equals("PHYSTEMP")))
+          || StanfordIndexer.CLOSED_LIBS.contains(library))
```

Comment:
Oh, yes. I've got my Atom configured to insert 2 spaces when I hit tab but it seems all the previous code uses tabs. Is there an easy way to convert the tabs to spaces?